test/10_rsk_owner_ownerOf.js

```diff
@@ -0,0 +1,97 @@
+const RNS = artifacts.require('RNS');
+const Token = artifacts.require('ERC677TokenContract');
+const TokenRegistrar = artifacts.require('TokenRegistrar');
+const RSKOwner = artifacts.require('RSKOwner');
+const FIFSRegistrar = artifacts.require('FIFSRegistrar');
+const NamePrice = artifacts.require('NamePrice');
+
+const namehash = require('eth-ens-namehash').hash;
+const expect = require('chai').expect;
+const helpers = require('@openzeppelin/test-helpers');
+
+contract('FIFS Registrar - overrided ownerOf', async (accounts) => {
+  let rns, token, tokenRegistrar, rskOwner, fifsRegistrar, namePrice;
+  const pool = accounts[6];
+
+  beforeEach(async () => {
+    const rootNode = namehash('rsk');
+
+    rns = await RNS.new();
+    token = await Token.new(accounts[0], web3.utils.toBN('1000000000000000000000'));
+    tokenRegistrar = await TokenRegistrar.new(rns.address, rootNode, token.address);
+
+    const migrationPeriod = web3.utils.toBN('1296000'); // 15 days
+    rskOwner = await RSKOwner.new(
+      tokenRegistrar.address,
+      migrationPeriod,
+      rns.address,
+      rootNode,
+    );
+    await rns.setSubnodeOwner('0x00', web3.utils.sha3('rsk'), rskOwner.address);
+
+    await helpers.time.increase(1296001);
+    namePrice = await NamePrice.new();
+    fifsRegistrar = await FIFSRegistrar.new(token.address, rskOwner.address, pool, namePrice.address);
+    await rskOwner.addRegistrar(fifsRegistrar.address, { from: accounts[0] });
+  });
+
+  describe('single name', async () => {
+
+    it('should not remove non expired name', async () => {
```

Comment:
Repeated test in 
https://github.com/rnsdomains/rns-rskregistrar/blob/fa6fe035c8b9a80556dcc87c82eb62825853fd48/test/5_rsk_owner_registration.js#L52